test/test-cases/multiple-routes/__snapshots__/multiple-routes.test.js.snap

```diff
@@ -19,25 +19,25 @@ SOURCE HTML:
     <link data-chunk="handlers-Home"
           rel="preload"
           as="script"
-          href="http://localhost:4004/handlers-Home-9299312b46da1275b24f.js"
+          href="http://localhost:4004/handlers-Home-d4afda529b3b1a8b8da7.js"
```

Comment:
huh?

Reply:
Because the script contents changed, it's changed the hash in the snapshot tests.